<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Login {
	background-color: rgb(247,247,247);
}

QFrame#loginForm {
	background-color: rgb(253,253,253);
    border: 1px solid #e0e0e0;
    border-radius: 8px;
}

QLabel#labelLogin {
    font-size: 24px;
    font-weight: bold;
    color: #1a365d;
	qproperty-alignment: AlignCenter;
}

QLabel#labelUsername, QLabel#labelPassword {
	font-size: 14px;
	font-weight: 500;
	color: #555555;
}

QPushButton#buttonLogin {
    background-color: rgb(45,45,45);
    color: rgb(253,253,253);
    font-size: 16px;
    font-weight: 500;
    border: none;
    border-radius: 4px; 
}

QPushButton#buttonLogin:hover {
	background-color: rgba(45,45,45, .90);
}

QLineEdit {
    color: #1a365d;
	padding: 8px;
    background-color: rgb(253,253,253);
    border: 1px solid #a0aec0;
    border-radius: 4px;
}

QLineEdit:focus {
    border: 2px solid #3182ce;
}

QCheckBox {
    font-size: 14px;
    color: rgb(44, 82, 130);
}

QCheckBox:hover {
    font-size: 14px;
    color: rgba(44, 82, 130,0.9);
}

QLabel#labelForgotPass {
    font-size: 14px;
    color: #e53e3e;
}

QLabel#labelForgotPass:hover {
    color: #c53030;
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>360</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="loginForm">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>340</width>
       <height>320</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>360</width>
       <height>320</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QLabel" name="labelLogin">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonLogin">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::FocusPolicy::StrongFocus</enum>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>298</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::FocusPolicy::StrongFocus</enum>
         </property>
         <property name="text">
          <string>Remember me</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelForgotPass">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::FocusPolicy::TabFocus</enum>
         </property>
         <property name="text">
          <string>Forgot password?</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>281</width>
        <height>65</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labelPassword">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditPassword">
         <property name="maximumSize">
          <size>
           <width>280</width>
           <height>40</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::FocusPolicy::StrongFocus</enum>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::EchoMode::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>281</width>
        <height>65</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labelUsername">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditUsername">
         <property name="maximumSize">
          <size>
           <width>280</width>
           <height>40</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::FocusPolicy::StrongFocus</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>360</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditUsername</tabstop>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>labelForgotPass</tabstop>
  <tabstop>buttonLogin</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
